### ISSUE TITLE
Problem using include with vriable for dynamic output

### ISSUE THREAD

**Issue by di5abled** [pixotech/labcoat#5]
I try to make a dynamic layout component which includes the relevant pattern based on a variable, like showen below. The problem is that Labcoat tries to include a pattern with the name "block" instead of parsing the variable property. 

This error only occures inside of labcoat, when I use this syntax with default twig this is no problem.

```
{% for block in page.layout %}
      {% include block.layout with block %}
{% endfor %}
```

**Comment by mattsharkey** [comment]
I made a quick test case for this issue, but I can't reproduce the problem on my machine. Would you mind running the test on your end and letting me know the results? If you've got Composer and PHPUnit installed, you should be able to unzip the file and run the `run-test.sh` script.

If the test passes, you might take a look inside the test case and make sure that it accurately reflects the way you're using Labcoat. If you can send me back a failing test, it will help out greatly.

Thanks!

[issue005.zip](https://github.com/pixotech/labcoat/files/161561/issue005.zip)

**Comment by di5abled** [comment]
Thank you, I'll do that asap :+1:

**Comment by di5abled** [comment]
Hy, the problem happens when the Patternlab itself is generated.
When I only use the Patternlab Loader like in your test I have no problems too.

I'll try to make a working test, but maybe you are the better one for this because I don't really have that much insights in how you generate the Patternlab output.

**Comment by di5abled** [comment]
Ok here's a failing test. We have a kind of unique setup and usage of labcoat, maybe we are missing something in our custom configs or something else? 

[issue005.zip](https://github.com/pixotech/labcoat/files/163698/issue005.zip)

**Comment by mattsharkey** [comment]
Thanks for that test. It helped immensely.

The error isn't being thrown when the template renders, but when the style guide generator is gathering information about template lineages, i.e. which templates include which. It doesn't know anything about dynamic includes, so it fails when it doesn't find a template with the same name as your variable.

I'll get to work on a fix that allows the generator to fail more gracefully under these circumstances. You still won't see any dynamic includes in your lineages, but it should allow the generator to complete successfully.

Thanks again for your help!

**Comment by di5abled** [comment]
oh, so you mean this will never work inside of the generator for patternlab? oh :-(

**Comment by mattsharkey** [comment]
You'll be able to generate Pattern Lab style guides, but you won't see accurate lineage information for templates that are loaded dynamically. (There's some information about lineages halfway down [this page](http://dmolsen.com/2013/11/14/pattern-lab-gets-annotations-code-view-and-more/).)

**Comment by mattsharkey** [comment]
Can you try replacing `vendor/pixo/labcoat/src/Styleguide/Styleguide.php` with the file in the attached zip and see if you're able to generate a style guide?

[Styleguide.php.zip](https://github.com/pixotech/labcoat/files/163866/Styleguide.php.zip)

**Comment by di5abled** [comment]
It is working now, thank you! :)
Would be great if you could release the fix soon, this blocks me a little bit on my current project :)

**Comment by mattsharkey** [comment]
This fix is available in v1.0.2.
